rclcpp/test/topic_statistics/test_subscription_topic_statistics.cpp

```diff
@@ -72,23 +122,25 @@ class EmptySubscriber : public rclcpp::Node
   EmptySubscriber(const std::string & name, const std::string & topic)
   : Node(name)
   {
+    // manually enable topic statistics via options
+    auto options = rclcpp::SubscriptionOptions();
+    options.topic_stats_options.state = rclcpp::TopicStatisticsState::Enable;
+
     auto callback = [this](Empty::UniquePtr msg) {
         this->receive_message(*msg);
       };
     subscription_ = create_subscription<Empty,
         std::function<void(Empty::UniquePtr)>>(
       topic,
       rclcpp::QoS(rclcpp::KeepAll()),
-      callback);
+      callback,
+      options);
   }
 
-  virtual ~EmptySubscriber() = default;
```

Comment:
If you remove this, then it is good practice to mark the class as `final`, because it is no longer safe to inherit from it generally, though it may still be in specific cases.

Reply:
This is a mistake, will revert.